ThunkLibs/include/common/Host.h

```diff
@@ -405,6 +405,12 @@ struct host_to_guest_convertible {
   {
     return {static_cast<uint32_t>(from.data)};
   }
+
+  // libGL also needs to allow long->int conversions for return values...
```

Comment:
I need to double-check if this is still required. The other changes in Host.h may have obsoleted it.